### DISCUSSION TITLE
problem with completion of course

### DISCUSSION THREAD

**Discussion by Aryaajain119** [github/india#383] [General]
guys, lets make an unofficial whatsapp grp because it is getting very difficult to communicate here 
even i am having problem in completing the course as it is very confusing where to start and how to go with it.

**Comment by TheSuranaverse** [comment]
@Aryaajain119 
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
@Aryaajain119 wp group is not preferred bcz no one wants to share their personal number on an unofficial grp of random people. That's where **Discord & Telegram** comes & trust me since many of you are not familiar with it, but once u r, WhatsApp would become boring. Also wp has a limit, whereas there's no limit on Discord ;)
ps- Developers & Tech-Enthusiasts will always prefer Discord/Telegram/Slack over wp ;)
That's the benefit of community.

**Comment by Aryaajain119** [comment]
I am not able to go ahead with the internship this is very confusing on
this part as we have completed the course but it's still showing in
progress.... if possible please share a contact no. So that we can
communicate easily as it gets difficult to explain the hardship we are
facing on mail rather it becomes much easier to explain on call....
The problem I m facing is I started with learning on the FRT Website > self
learning session> Azure Fundamentals (AZ-900)Duration 10 hrs 21 min >
describe core azure concepts (in progress)... pic1 and 2 what to to next it
still showing pending
